src/example.cpp

```diff
@@ -185,5 +198,8 @@ int main(int argc, const char* argv[])
 {
   parse(argc, argv);
 
+  // Parse argv as string
+  parse(argc, argv, true);
```

Comment:
I don't think adding all of this to the example is necessary. I would rather just have a test case that calls the new function.